modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/SwiftCodegen.java

```diff
@@ -250,6 +251,16 @@ public CodegenProperty fromProperty(String name, Property p) {
     return codegenProperty;
   }
 
+    public String toSwiftyEnumName(String value) {
+        // Prevent from breaking properly cased identifier
+        if (value.matches("[A-Z][a-z0-9]+[a-zA-Z0-9]*")) {
+            return value;
+        }
+        char[] separators = {'-', '_', ' '};
+        return WordUtils.capitalizeFully(StringUtils.lowerCase(value), separators).replaceAll("[-_ ]", "");
+    }
```

Comment:
There are 2 methods that can help you sanitize and camelize the Swift Enum name:

https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java#L1662

https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java#L1549

Here is how CSharp uses these 2 methods for variable naming.

https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/CSharpClientCodegen.java#L148